tests/programs/clinical-submission.js

```diff
@@ -106,32 +105,7 @@ module.exports = {
     browser.expect.element('#button-clear-submission').to.have.attribute('disabled');
   },
 
-  // 'Submission - Upload Bad Clinical Data and Clear Selected Files': browser => {
-  //   startAsUser(browser)(TEST_USERS.DCC_ADMIN)
-  //     .perform(async () => {
-  //       await submitClinicalData({
-  //         jwt: TEST_USERS.DCC_ADMIN.token,
-  //         shortName: program.shortName,
-  //         good: 'false',
-  //       });
-  //     })
-  //     .pause(2500)
-  //     .url(buildUrl(`/submission/program/${program.shortName}/clinical-submission`));
-  //   browser.expect.element('#button-validate-submission').to.have.attribute('disabled');
-  //   browser.expect.element('#button-submission-sign-off').to.have.attribute('disabled');
-  //   browser.expect.element('#button-clear-submission').to.not.have.attribute('disabled');
-  //   browser.assert.visible('#button-clear-selected-file');
-
-  //   browser.click('#button-clear-selected-file').pause(2000);
-
-  //   // Add logic here
-
-  //   browser.expect.element('#button-validate-submission').to.not.have.attribute('disabled');
-  //   browser.expect.element('#button-submission-sign-off').to.have.attribute('disabled');
-  //   browser.expect.element('#button-clear-submission').to.not.have.attribute('disabled');
-  // },
```

Comment:
this is not needed? can we just take it out?

Reply:
I believe this is already taken out.

Reply:
oh oops my bad, had a wrong github setting that messed with my reading :P sorry!